README.md

```diff
@@ -47,13 +47,17 @@ After you have installed the CLI, you can run the following from the root of the
 ```bash
 diesel migration run --database-url <DATABASE_URL>
 ```
+Where ```<DATABASE_URL>``` uses the ```postgess://``` URI Example:
```

Comment:
This is markdown, so we can use single backticks here for this line instead of triple. It'll do a smaller code representation and look better when rendered.